CAM/StopGameplayCamShaking.md

```diff
@@ -5,10 +5,10 @@ ns: CAM
 
 ```c
 // 0x0EF93E9F3D08C178 0xFD569E4E
-void STOP_GAMEPLAY_CAM_SHAKING(BOOL p0);
+void STOP_GAMEPLAY_CAM_SHAKING(BOOL stopImmediately);
 ```
 
 
 ## Parameters
-* **p0**: 
+* **stopImmediately**: Should the shake stop this frame?
```

Comment:
It stops immediately, yes (you can remove the question mark 😅). It's preferable to name this `bStopImmediately` since it's a `BOOL` parameter.